### ISSUE TITLE
docker run hangs when R_PACKAGES is specified and R_PACKAGES_GH is not specified

### ISSUE THREAD

**Issue by skadauke** [skadauke/rsp-train#2]
Reprex:

```
docker run \
    -e PW_SEED=12345 \
    -e GH_REPO=https://github.com/skadauke/intro-to-r-for-clinicians-chop \
    -e R_PACKAGES=shiny \
    -e R_PACKAGES_GH=rstudio/DT \
    skadauke/rsp-train
``` 
This works, but when we remove `-e R_PACKAGES_GH=rstudio/DT \`, it hangs before docker produces any output.

**Comment by skadauke** [comment]
This seems to be related to `start_rsp_train.sh` not parsing arguments correctly.

**Comment by skadauke** [comment]
This was addressed in PR #3
